src/extra/Form/markdown.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { FormWidgetProps } from 'rendition/dist/unstable';
+import { Link, Textarea, Theme } from '../../';
+import { Form } from '../../unstable';
+import { TabbedBox } from '../../unstable/components/Form/widgets/TabbedBox';
+import { Markdown } from '../Markdown';
+
+class Widget extends React.Component<FormWidgetProps, {}> {
+	public render() {
+		// Note: since React 15.2.0 we can't forward unknown element attributes, so we
+		// exclude the "options" and "schema" ones here.
+		const {
+			value,
+			readonly,
+			disabled,
+			autofocus,
+			onBlur,
+			onFocus,
+			options,
+			schema,
+			formContext,
+			...inputProps
+		} = this.props;
+
+		const change = ({ target: { value } }: any) => {
+			return this.props.onChange(value === '' ? options.emptyValue : value);
+		};
+
+		return (
+			<TabbedBox
+				tabs={['Write', 'Preview']}
+				help={
+					<Link
+						fontSize={2}
+						href={'https://guides.github.com/features/mastering-markdown/'}
+						blank
+					>
+						About markdown
+					</Link>
+				}
+			>
+				<Textarea
+					placeholder="This text supports markdown"
+					readOnly={readonly}
+					disabled={disabled}
+					autoFocus={autofocus}
+					value={value || ''}
+					{...inputProps}
+					onChange={change}
+					onBlur={
+						onBlur &&
+						((event: any) => onBlur(inputProps.id, event.target.value))
```

Comment:
How about `React.FocusEvent<HTMLTextAreaElement>` instead of `any`?
PS: `onBlur?: FocusEventHandler<T>;` so that should be the correct event.